### PR TITLE
#12537 issue solution

### PR DESCRIPTION
**Pull request by yoon-hyoung**
* printing prefix message by parameter value

### CODE REVIEW

File: tensorflow/python/ops/logging_ops.py [hunk 1]
```diff
@@ -124,7 +125,7 @@ def image_summary(tag, tensor, max_images=3, collections=None, name=None):
   """Outputs a `Summary` protocol buffer with images.
 
   For an explanation of why this op was deprecated, and information on how to
-  migrate, look ['here'](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/deprecated/__init__.py)
+  migrate, look ['here'](https://www.tensorflow.org/code/tensorflow/contrib/deprecated/__init__.py)
```
Comment (byronyi) [inline tensorflow/python/ops/logging_ops.py]: This link appears to be broken. I do not understand why you would like to change the links here.

Comment (byronyi) [inline tensorflow/python/ops/logging_ops.py]: This link appears to be broken. I do not understand why you would like to change the links here.

**Review by vrv — COMMENTED** [review]
I don't think this works, message defaults to None and so the flag doesn't do anything different.

The prefix comes from the C++ LOG(INFO) statement.  One would have to find a way to prevent the prefix from being printed there, probably via an environment variable.

There are also unrelated changes in this PR, which you would want to remove.

**Comment by drpngx** [comment]
Closing, since it's not clear that this achieves the desired result. Feel free to cleanup and reopen with comments. Thanks!

**Pull request closed without merging**